pages/basics/content/zh/dynamic-styles.md

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
The content for this file has changed a little bit (specifically the example: https://raw.githubusercontent.com/kentcdodds/glamorous-website/master/pages/basics/content/dynamic-styles.md)